### PR TITLE
Initial mako config

### PR DESCRIPTION
**Pull request by Lyndeno**
Mako support in Stylix.

### CODE REVIEW

**Comment by Lyndeno** [comment]
Not ready, some things to be fixed

**Comment by Lyndeno** [comment]
Oops nvm, had the wrong config enabled. This is good to go

File: modules/mako.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{pkgs, config, lib, ... }:
+
+with config.lib.stylix.colors.withHashtag;
+with config.stylix.fonts;
+{
+  # Referenced https://github.com/stacyharper/base16-mako
+  home-manager.sharedModules = [{
+    programs.mako = {
+      backgroundColor = base00;
+      borderColor = base0D;
```
Comment (danth) [inline modules/mako.nix:10]: Do we need to set `progressColor` too?

Comment (Lyndeno) [reply modules/mako.nix:10]: Probably a good idea. I forgot that option existed.

**Comment by Lyndeno** [comment]
I set the progressColor option. This should be ready now.

**Comment by danth** [comment]
Great, thanks!

Apologies for the failing check. It wasn't meant to run for pull requests.

**Pull request merged**